Client crash (every 3-4 games)

Since 0.9.x, client crashes on me quite often.
Game stays up, but since client crashes, I lose conn to all players.

Here is the Java crash log:
https://gist.github.com/tomaskir/c88e45eea3acdb3a17d80ae10084bbcd

Script would not launch as provided.
Had to adjust line 5 for `%java_binary%` to be enclosed in `"` (since the path contains spaces).

After that fix, I now get:
```
Error: Could not find or load main class FAF
Caused by: java.lang.ClassNotFoundException: FAF
```

--- a/src/main/java/com/faforever/client/chat/AbstractChatTabController.java
+++ b/src/main/java/com/faforever/client/chat/AbstractChatTabController.java
@@ -140,7 +140,7 @@ public abstract class AbstractChatTabController implements Controller<Tab> {
   private String receiver;
   private Pattern mentionPattern;
   private ChatMessage lastMessage;
-  private WebEngine engine;
+  WebEngine engine;
 
   @Inject
   // TODO cut dependencies

--- a/src/main/java/com/faforever/client/chat/ChannelTabController.java
+++ b/src/main/java/com/faforever/client/chat/ChannelTabController.java
@@ -364,8 +364,8 @@ public class ChannelTabController extends AbstractChatTabController {
     if (cssClass.isEmpty()) {
       return;
     }
-    Platform.runLater(() -> getJsObject().call("removeUserMessageClass", String.format(USER_CSS_CLASS_FORMAT, chatUser.getUsername()), cssClass));
-
+    //Workaround for issue #1080 https://github.com/FAForever/downlords-faf-client/issues/1080
+    Platform.runLater(() -> engine.executeScript("removeUserMessageClass(\'" + String.format(USER_CSS_CLASS_FORMAT, chatUser.getUsername()) + "\',\'" + cssClass + "\');"));
   }
 
   private void addUserMessageClass(ChatChannelUser player, String cssClass) {

